builtin/settingtypes.txt

```diff
@@ -475,16 +475,20 @@ enable_bloom_debug (Enable Bloom Debug) bool false
 #    on certain GPUs and video drivers.
 enable_bloom_dedicated_texture (Enable Bloom Dedicated Texture) bool false
 
-#    Set the intensity of bloom
+#    Defines how much bloom is applied to the rendered image
 #    Smaller values make bloom more subtle
 #    Range: from 0.01 to 1.0, default: 0.05
 bloom_intensity (Bloom Intensity) float 0.05 0.01 1.0
 
+#    Defines the magnitude of bloom overexposure.
+#    Range: from 1.0 to 10.0, default: 1.0
+bloom_strength_factor (Bloom Strength Factor) float 1.0 0.1 10.0
+
 #    Set the radius of the bloom filter in pixels.
 #    Larger values render more glow around bright objects
 #    at the cost of higher resource consumption.
-#    Range: from 1 to 64, default: 16
-bloom_radius (Bloom Radius) int 16 1 64
+#    Range: from 0.1 to 8, default: 1
+bloom_radius (Bloom Radius) float 1 0.1 8
```

Comment:
It's not clear to me what a < 1 radius means. If that is scaled by something we should mention it here.